packages/backend/src/clients/EmailClient/templates/imageNotification.html

```diff
@@ -396,40 +355,68 @@
-                                                    class="logo"
-                                                    alt="lightdash logo"
-                                                    src="{{host}}/lightdash-logo.png"
-                                                />
-
-                                                <h2>{{title}}</h2>
-                                                <p>{{description}}</p>
-
+                                                <p class="description">
+                                                    {{date}}
+                                                </p>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td class="card">
                                                 <img
                                                     alt="chart or dashboard image"
                                                     src="{{imageUrl}}"
+                                                    class="chart dashboard"
                                                 />
+                                            </td>
+                                        </tr>
 
-                                                <a href="{{url}}" class="btn"
-                                                    >View in Lightdash</a
+                                        <tr>
+                                            <td class="btn-container">
+                                                <button
+                                                    href="{{url}}"
+                                                    class="btn-primary"
                                                 >
-                                                <!-- START FOOTER -->
+                                                    View in Lightdash
+                                                </button>
+                                            </td>
+                                        </tr>
+                                        <tr>
+                                            <td>
                                                 <div class="footer">
                                                     <table
                                                         role="presentation"
                                                         border="0"
                                                         cellpadding="0"
                                                         cellspacing="0"
                                                     >
+                                                        <tr>
+                                                            <td
+                                                                class="content-block"
+                                                            >
+                                                                <a
```

Comment:
Are we missing a href in this link?

Reply:
yes, don't know what it should be pointing to